code/memanalyzer.py

```diff
@@ -120,7 +123,8 @@ def modules_get():
     del modules_['NETBIOS']
     return modules_
 
-try:
+
```

Comment:
`try: except: raise` is the same as the default behaviour.   The code will behave the same without it.   I removed it and replaced it with a `if __name__ == '__main__':` in the case any of the functions would end up getting reused.